spread/jsonunit.go

```diff
@@ -0,0 +1,168 @@
+package spread
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+)
+
+type JSONUnitTestSuites struct {
+	Suites   []*JSONUnitTestSuite
+}
+
+func (tss *JSONUnitTestSuites) addSuite(suite *JSONUnitTestSuite) {
+	tss.Suites = append(tss.Suites, suite)
+}
+
+func (tss *JSONUnitTestSuites) getSuite(suiteName string) *JSONUnitTestSuite {
+    for _, s := range tss.Suites {
+        if s.Name == suiteName {
+            return s
+        }
+    }
+	suite := NewJSONUnitTestSuite(suiteName)
+
+	tss.addSuite(suite)	
+	return suite
+}
+
+type JSONUnitTestSuite struct {
+	Successfull int              `json:"successfull,attr"`
+	Failed      int              `json:"failed,attr"`
+	Aborted     int              `json:"aborted,attr"`
+	Total       int              `json:"total,attr"`
+	Time        string           `json:"time,attr"`
+	Name        string           `json:"name,attr"`
+	TestCases   []*JSONUnitTestCase
+}
+
+func NewJSONUnitTestSuite(suiteName string) *JSONUnitTestSuite {
+	return &JSONUnitTestSuite{
+		Successfull: 0,
+		Failed: 0,
+		Aborted: 0,
+		Total: 0,
+		Time: "",
```

Comment:
You don't need these initializations, they are already the default ones.

Reply:
nice, thanks